supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigBackendFileHashing.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.backend.impl.sfs.config
+
+import java.io.{File, FileInputStream}
+
+import akka.event.LoggingAdapter
+import cromwell.backend.callcaching.FileHasherWorkerActor.SingleFileHashRequest
+
+import scala.util.{Failure, Success, Try}
+
+private[config] object ConfigBackendFileHashing {
+  def getMd5Result(request: SingleFileHashRequest, log: LoggingAdapter) = {
+    val md5Result = for {
+      fileInputStream <- Try(new FileInputStream(new File(request.file.valueString)))
+      // Use = here so that closeAndLog runs even if this fails:
+      md5String = Try(org.apache.commons.codec.digest.DigestUtils.md5Hex(fileInputStream))
+      _ = closeAndLog(fileInputStream, log)
```

Comment:
I think `recover`/`recoverWith` might be a more straightforward way of handling this